lib/elixir/lib/calendar.ex

```diff
@@ -804,6 +805,16 @@ defmodule Calendar do
     parse(rest, datetime, format_options, [result | acc])
   end
 
+  # Epoch time
+  defp format_modifiers("s" <> rest, width, pad, datetime, format_options, acc) do
+    result = datetime
+    |> NaiveDateTime.to_erl()
+    |> :calendar.datetime_to_gregorian_seconds
```

Comment:
Shouldn't we also support DateTime? Also, please implement this on top of NaiveDateTime functions themselves. Probably `NaiveDateTime.diff(..., ~N[1970-01-01 00:00:00])` or similar?

Reply:
I'm still fairly new to Elixir. `NaiveDateTime.diff` is much simpler. I've made that change. Thank you!

> Shoudn't we also support DateTime?

I'm not following what you mean by this.

Reply:
We also have a `DateTime` module (and struct) and we support both as inputs to `strftime`. :)

Reply:
They both should work. I added a test for it.

Reply:
The problem is that the current implementation is discarding/ignoring the timezone offset. :( Try adding some timezone offset to the DateTime. The result should change but I think it will be the same. Also, please remember to run `bin/mix format`. :)